src/main/java/io/github/thebusybiscuit/slimefun4/api/ErrorReport.java

```diff
@@ -26,10 +28,15 @@
 import me.mrCookieSlime.Slimefun.api.Slimefun;
 
 public class ErrorReport {
-	
+
+	private static final SimpleDateFormat sdf = new SimpleDateFormat("yyyy-MM-dd_HH-mm-ss");
+	private static final List<String> errors = new ArrayList<>();
```

Comment:
I have a CS-CoreLib2 implementation for that, CappedLinkedList or so. It auto-removes elements already.

Reply:
Ah cool!